pkgs/by-name/co/concourse/elm-srcs.nix

```
@@ -0,0 +1,157 @@
+{
+
+  "NoRedInk/elm-simple-fuzzy" = {
+    sha256 = "17nzrwaax86db0fg6sdiqzsrkmsczdi9q3xmp7i7yinin42b52dq";
+    version = "1.0.3";
+  };
+
+  "elm-community/graph" = {
+    sha256 = "1rwsq2126q0rb4vmy95ajxfm3m063d6lw0p90d510nzcrbm9bxbc";
+    version = "6.0.0";
+  };
+
```

```suggestion
  "NoRedInk/elm-simple-fuzzy" = {
    sha256 = "17nzrwaax86db0fg6sdiqzsrkmsczdi9q3xmp7i7yinin42b52dq";
    version = "1.0.3";
  };
  "elm-community/graph" = {
    sha256 = "1rwsq2126q0rb4vmy95ajxfm3m063d6lw0p90d510nzcrbm9bxbc";
    version = "6.0.0";
  };
```
Would be nice if we could exclude the new lines and use hash with sri hashes

---

I'll delete all the newlines, but we cannot replace `sha256` with SRI hashes. This is because the `elm-srcs.nix` file feeds into
```nix
    postConfigure = elmPackages.fetchElmDeps {
      elmPackages = import ./elm-srcs.nix;
      elmVersion = elmPackages.elm.version;
      registryDat = ./registry.dat;
    };
```
and `makeDotElm.nix` explicitly requests a `sha256` field.